aep/general/0159/aep.md.j2

```diff
@@ -1,5 +1,69 @@
 # Reading across collections
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/13) for more
-information.
+Sometimes, it is useful for a user to be able to retrieve resources across
+multiple collections, or retrieve a single resource without needing to know
+what collection it is in.
+
+## Guidance
+
+APIs **may** support reading resources across multiple collections by allowing
+users to specify a `-` (the hyphen or dash character) as a wildcard character
+in a standard [`List`](./list) method:
+
+```
+GET /v1/publishers/-/books?filter=...
+```
+
+- The URI pattern **must** still be specified with `*` and permit the
```

Comment:
Is this the URI pattern specified in the `google.api.resource` annotation? An example here would be helpful

Reply:
Yep; moving this to a protobuf tab.